src/components/OrderBookChart.stories.tsx

```diff
@@ -0,0 +1,249 @@
+
+type SampleData = typeof sampleDataSet[0]
+
+const networkIds = Object.values(Network).filter(Number.isInteger)
+
+const defaultNetworkId = Network.Mainnet
+// All Default Tokens
+const tokenList: TokenDetails[] = CONFIG.initialTokenList.map(
+  ({ id, name, symbol, addressByNetwork, decimals = 18 }) => ({
+    id,
+    name,
+    symbol,
+    address: addressByNetwork[defaultNetworkId] || '0x',
+    decimals: decimals,
+  }),
+)
+
+// Default params to be used as initial values
+// and when there's no Token found for symbol
+const defaultParams: Omit<OrderBookChartProps, 'data'> = {
+  baseToken: {
+    id: 1,
+    name: 'Base Token',
+    symbol: 'BASE',
+    address: '0x1',
+    decimals: 18,
+  },
+  quoteToken: {
+    id: 2,
+    name: 'Quote Token',
+    symbol: 'QUOTE',
+    address: '0x2',
+    decimals: 18,
+  },
+  networkId: defaultNetworkId,
+}
+
+// Token symbols to use in control selector
+const configTokenSymbols = [defaultParams.baseToken, defaultParams.quoteToken, ...tokenList].map(
+  (token) => token.symbol || token.address,
+)
+
+export default {
+  title: 'Components/OrderBookChart',
+  component: OrderBookChart,
+  argTypes: {
+    networkId: { control: { type: 'inline-radio', options: networkIds } },
+    baseToken: {
+      control: { type: 'select', options: configTokenSymbols },
+    },
+    quoteToken: {
+      control: { type: 'select', options: configTokenSymbols },
+    },
+    name: { control: null },
+    description: { control: null },
+    data: { control: null },
+  },
+  decorators: [(Story): JSX.Element => <div style={{ height: '97vh' }}>{Story()}</div>],
+} as Meta
+
+//-------------------REAL-DATA------------------
+
+// Template for real data
+// fetched from price-estimation service
+const RealTemplate: Story<OrderBookChartProps> = (args) => <OrderBookChart {...args} />
+
+const DAI = tokenList.find((token) => token.symbol === 'DAI')
+const USDC = tokenList.find((token) => token.symbol === 'USDC')
+export const USDC_DAI = RealTemplate.bind({})
+USDC_DAI.args = {
+  ...defaultParams,
+  baseToken: DAI || tokenList[0],
+  quoteToken: USDC || tokenList[1],
+  data: processRawApiData({ data: realData, baseToken: DAI || tokenList[0], quoteToken: USDC || tokenList[1] }),
+}
+USDC_DAI.argTypes = {
+  baseToken: { control: null },
+  quoteToken: { control: null },
+  data: { control: null },
+}
+
+//-------------------SAMPLE-DATA------------------
+
+interface ProduceOrderBookArgsParams {
+  sampleData: Pick<SampleData, 'asks' | 'bids'>
+  orderbookProps: Omit<OrderBookChartProps, 'data'>
+}
+// normalizing sample data
+// because real data is slightly more complex
+const produceOrderBookProps = ({ sampleData, orderbookProps }: ProduceOrderBookArgsParams): OrderBookChartProps => {
+  const { baseToken, quoteToken } = orderbookProps
+
+  const { asks = [], bids = [] } = sampleData
+  const normalizedRawData: RawApiData = {
+    asks: asks.map(({ amount, price }) => ({
```

Comment:
Amount should be afaik an string, not a number. They were changing that in the backend, not sure if they did it already. We should try to operate with Bignumbers anyways, so we don't make float calculating errors, and so that change from the backend becomes transparent to us.

Reply:
https://dex-price-estimator.gnosis.io/api/v1/markets/7-4?atoms=true&hops=2 returns numbers only

Reply:
My point is, it should be string. If they send it in number, we should convert it to string before operating, right? 
On top of preventing calculation issues, it will make the change in the backend trivial for us.

Maybe is not for this PR if it's not a trivial change..

Reply:
they do get turn into BigNumber inside `processData`